driver/build/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM tutum/dnsutils
```

Comment:
Can we use more standard base image? Like ubuntu. Tutum is Docker Container Cloud.